system/jlib/jstats.cpp

```diff
@@ -963,6 +973,7 @@ static const constexpr StatisticMeta statsMetaData[StMax] = {
     { NUMSTAT(AckRetries), "The number of times the server failed to receive a response from an agent within the expected time" },
     { SIZESTAT(ContinuationData), "The total size of continuation data sent from agent to the server\nA large number may indicate a poor filter, or merging from many different index locations" },
     { NUMSTAT(ContinuationRequests), "The number of times the agent indicated there was more data to be returned" },
+    { NUMSTAT(Failures), "The number of times a query has failed" },
```

Comment:
I didn't realize these were here, I'll look through them and see if we can publish the relevant ones in the JTrace readme